controllers/userController.js

```diff
@@ -0,0 +1,43 @@
+const keywordModel = require("../models/keywordModel");
+const userModel = require("../models/userModel");
+const { isValidString } = require("../utils/validation");
+
+const create = async (req, res) => {
+  if (!isValidString(req.body.uid)) {
+    return res.status(400).send({ message: "[InvalidUid] Error occured" });
+  }
+  if (!isValidString(req.body.email)) {
+    return res.status(400).send({ message: "[InvalidEmail] Error occured" });
+  }
+  if (!isValidString(req.body.displayName)) {
+    return res.status(400).send({ message: "[InvalidDisplayName] Error occured" });
+  }
+  if (!isValidString(req.body.photoURL)) {
+    return res.status(400).send({ message: "[InvalidPhotoURL] Error occured" });
+  }
```

Comment:
```suggestion
  if (!isValidString(req.body.uid) || isBlank(req.body.uid)) {
    return res.status(400).send({ message: "[InvalidUid] Error occured" });
  }
  if (!isValidString(req.body.email) || isBlank(req.body.email)) {
    return res.status(400).send({ message: "[InvalidEmail] Error occured" });
  }
  if (!isValidString(req.body.displayName) || isBlank(req.body.displayName)) {
    return res.status(400).send({ message: "[InvalidDisplayName] Error occured" });
  }
  if (!isValidString(req.body.photoURL) || isBlank(req.body.photoURL)) {
    return res.status(400).send({ message: "[InvalidPhotoURL] Error occured" });
  }
```


`isValidString` 에는 빈 문자열인 것은 검사하고 있지 않아서, validation 파일 내에 빈 문자열 여부를 확인하는 `isBlank` 함수로 더블 체크를 해야할 것 같습니다.